pkg/injector/annotations/annotations.go

```diff
@@ -67,6 +67,7 @@ const (
 	KeyPluggableComponentsSocketsFolder = "dapr.io/pluggable-components-sockets-folder"
 	KeyPluggableComponentContainer      = "dapr.io/component-container"
 	KeyPluggableComponentsInjection     = "dapr.io/inject-pluggable-components"
+	KeyAppChannel                       = "dapr.io/app-channel-address"
```

Comment:
Does this have any real life application? If this is added, then the app has a Deployment in K8s to be talked via localhost, not?

Reply:
Yes, this tells Dapr to send events and messages to this configured address which is not localhost